traffic_ops/testing/api/v13/cdn_domains_test.go

```diff
@@ -0,0 +1,42 @@
+package v13
+
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"testing"
+
+	log "github.com/apache/trafficcontrol/lib/go-log"
```

Comment:
The alias isn't necessary `"github.com/apache/trafficcontrol/lib/go-log"` automatically imports as `log`